### ISSUE TITLE
Contacts app FC's

### ISSUE THREAD

**Issue by sayjay09** [loxK/android_device_htc_hero#17]
The contact app FC's on open

**Comment by cbruegg** [comment]
Logcat?

**Comment by sayjay09** [comment]
Without sounding like a retard, i do the logcat and reproduce the issue but how to i save it as a .txt file?

**Comment by cbruegg** [comment]
adb logcat > log.txt ;-)

**Comment by sayjay09** [comment]
I hope this has the right info, i did the command above and then reproduced the steps. I dunno how to upload here but heres the uploaded file on mediafire. http://www.mediafire.com/?kwouwomymyg

The issue happens when the contacts tab in the contacts app and in the dialer is selected. Is there another app i can use in the meantime?

**Comment by loxK** [comment]
Try wiping.

**Comment by sayjay09** [comment]
I wiped, the app works before syncing contacts from my google account after the sync is complete, again the app fails.

**Comment by loxK** [comment]
Send ur logcat on pastebin please

**Comment by sayjay09** [comment]
Heres the link http://pastebin.com/7nGxvzXH

**Comment by sayjay09** [comment]
As a good replacement, i used the phonebook 2.0 app.

**Comment by vorcigernix** [comment]
Weird, mine works just fine. Are you reporting fc in contact app which comes with rom?

**Comment by sayjay09** [comment]
Yes, this is the stock contacts app it works fine before google contacts sync which is setup in accounts and sync. Once it has synced contacts tab of the contacts app fails

**Comment by petsasj** [comment]
I can confirm this happens as well. Wiped everything before installing.

**Comment by vorcigernix** [comment]
Looks like null pointer exception with lot of errors when drawing contact list (there are no db errors I think). What is your configuration? You have all contacts on google?

**Comment by sayjay09** [comment]
Yes all of my contacts are on google.

**Comment by vorcigernix** [comment]
anyone else with this configuration? I have 90% of contacts on exchange, so I can't look further into this without wiping (and I can't experiment much in daytime). Can you try to add some contact to sim? I think they are invisible by default anyway.

**Comment by misteriks** [comment]
"The issue happens when the contacts tab in the contacts app and in the dialer is selected."

This is correct, but only when the contact has multiple phone numbers and when there is no primary phone number selected. If you set a primary number and select the dialer then this issue doesn't occur.
This also doesn't happen with contacts that have only 1 phone number.

**Comment by vorcigernix** [comment]
nice work man

**Comment by petsasj** [comment]
indeed i can confirm that.
